spec/requests/api/admin/green_lanes/themes_controller_spec.rb

```diff
@@ -0,0 +1,30 @@
+RSpec.describe Api::Admin::GreenLanes::ThemesController do
+  subject(:page_response) { make_request && response }
+
+  before do
+    allow(TradeTariffBackend).to receive(:service).and_return 'xi'
+  end
+
+  let(:json_response) { JSON.parse(page_response.body) }
+  let(:theme) { create :green_lanes_theme }
+
+  describe 'GET to #index' do
+    let(:make_request) do
+      authenticated_get api_admin_themes_path(format: :json)
+    end
+
+    context 'with some themes' do
+      before do
+        theme
+      end
```

Comment:
super minor but this is so short it could just be a one liner `before { theme }` ? no strong feelings either way on this

Reply:
done